Docs: Agregar resumen del capitulo 8

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 8 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Docs: Agregar resumen del capitulo 8

## what

- Agregar el resumen del capitulo 8 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch8/Assets/HTML5.png:
Para los screenshots pueden ser solo del navegador para ver el resultado del html, ya que el código lo tenemos en git.

semanticmx reviewed (changes requested):
muy bien, solo hay que reemplazar los screenshots y colocar los del browser

IIKUYY committed:
Chores: Corregir archivos de assets

## what

- Eliminar screenshots del código y agregar el del navegador

## why

- Corregir error en los archivos y cambiarlos por los carrectos

## references

-

IIKUYY commented:
Cambios realizados

semanticmx reviewed (approved):
muy bien!
